syncmaster/schemas/v1/transfers/file_format.py

```diff
@@ -27,3 +32,9 @@ class JSON(BaseModel):
     type: JSON_FORMAT
     encoding: str = "utf-8"
     line_sep: str = "\n"
+
+
+class Excel(BaseModel):
+    type: EXCEL_FORMAT
+    include_header: bool
```

Comment:
CSV has option `header`, Excel `include_header`. This doesn't look consistent.
Also, what do you think about adding a default value?